src/main/java/net/dv8tion/jda/api/entities/Category.java

```diff
@@ -159,6 +159,43 @@
     @CheckReturnValue
     ChannelAction<VoiceChannel> createVoiceChannel(@Nonnull String name);
 
+    /**
+     * Creates a new {@link net.dv8tion.jda.api.entities.StageChannel StageChannel} with this Category as parent.
+     * For this to be successful, the logged in account has to have the
+     * {@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL MANAGE_CHANNEL} Permission in the {@link net.dv8tion.jda.api.entities.Guild Guild}.
```

Comment:
is this really true considering you can also allow this permission for permission holders for categories?